packages/contractkit/src/wrappers/MultiSig.ts

```diff
@@ -0,0 +1,34 @@
+import { TransactionObject } from 'web3/eth/types'
+import { Address } from '../base'
+import { MultiSig } from '../generated/types/MultiSig'
+import { BaseWrapper, proxyCall, stringToBytes, toTransactionObject } from './BaseWrapper'
+
+/**
+ * Contract for handling multisig actions
+ */
+export class MultiSigWrapper extends BaseWrapper<MultiSig> {
+  /**
+   * Allows an owner to submit and confirm a transaction.
```

Comment:
nit: Please expand on this. e.g.
```
If an unexecuted transaction matching `txObject` exists on the multisig, adds a confirmation to that tx ID. Otherwise, submits the `txObject` to the multisig.
```